enterprise/admin/views.py

```diff
@@ -42,6 +43,7 @@ class ContextParameters(object):
         LEARNERS = "learners"
         PENDING_LEARNERS = "pending_learners"
         MANAGE_LEARNERS_FORM = "manage_learners_form"
+        FILTERSTRING_NAME = "filterstring"
```

Comment:
I personally find the term "filterstring" to be confusing, and it took me a while to figure out what it actually was here. Though in your defence, it is a string that is used to filter the results. I just think that `search_keyword` or something like that would be much more clear.

Besides that, I think that calling it `FILTERSTRING_NAME` is very confusing. The value of `FILTERSTRING_NAME` is not the name of the search/filter string (the value is not `"q"`); the value is the search/filter string itself. So I would suggest naming this var `SEARCH_KEYWORD` instead of `FILTERSTRING_NAME`.